truffle-config.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
      host: "127.0.0.1",     // Localhost (default: none)
      port: 8545,            // Standard Ethereum port (default: none)
      network_id: "1",       // Any network (default: none)
+     from: "0x5E904A05033D728315dD23Bea210d17A68cA3d19",
```

Comment:
tested this again locally. I'm 99% sure this is not needed.

Reply:
Yes, you are correct. I ran through it again and it is no longer giving me the error